api/app/Generators/UserCsvGenerator.php

```diff
@@ -0,0 +1,153 @@
+        'citizenship',
+        'interested_in_languages',
+        'first_official_language',
+        'second_language_exam_completed',
+        'second_language_exam_validity',
+        'comprehension_level',
+        'writing_level',
+        'oral_interaction_level',
+        'estimated_language_ability',
+        'government_employee',
+        'department',
+        'employee_type',
+        'current_classification',
+        'priority_entitlement',
+        'priority_number',
+        'location_preferences',
+        'location_exemptions',
+        'accept_temporary',
+        'accepted_operational_requirements',
+        'woman',
+        'indigenous',
+        'visible_minority',
+        'disability',
+        'skills',
+    ];
+
+    public function __construct(protected array $ids, public string $fileName, public ?string $dir, protected ?string $lang = 'en')
+    {
+        parent::__construct($fileName, $dir);
+    }
+
+    public function generate(): self
+    {
+        $this->spreadsheet = new Spreadsheet();
+
+        $sheet = $this->spreadsheet->getActiveSheet();
+        $localizedHeaders = array_map(function ($key) {
+            return $this->localizeHeading($key);
+        }, $this->headerLocaleKeys);
+
+        $sheet->fromArray($localizedHeaders, null, 'A1');
+        $currentUser = 1;
+
+        User::with([
+            'department',
+            'currentClassification',
+            'userSkills' => ['skill'],
+        ])
+            ->whereIn('id', $this->ids)
+            ->chunk(200, function ($users) use ($sheet, &$currentUser) {
+                foreach ($users as $user) {
+
+                    $department = $user->department()->first();
+                    $preferences = $user->getOperationalRequirements();
+                    $indigenousCommunities = Arr::where($user->indigenous_communities, function ($community) {
+                        return $community !== IndigenousCommunity::LEGACY_IS_INDIGENOUS->name;
+                    });
+                    $userSkills = $user->userSkills->map(function ($userSkill) {
+                        return $userSkill->skill->name[$this->lang] ?? '';
+                    });
+
+                    $values = [
+                        $user->first_name, // First name
+                        $user->last_name, // Last name
+                        $this->localizeEnum($user->armed_forces_status, ArmedForcesStatus::class),
+                        $this->localizeEnum($user->citizenship, CitizenshipStatus::class),
+                        $this->lookingForLanguages($user),
+                        $this->localizeEnum($user->first_official_language, Language::class),
+                        is_null($user->second_language_exam_completed) ? '' : $this->yesOrNo($user->second_language_exam_completed), // Bilingual evaluation
+                        $this->yesOrNo($user->second_language_exam_validity),
+                        $user->comprehension_level, // Reading level
```

Comment:
Good catch, I did this in two because I initiall missed the fact your mentioned it was in both csv :sweat: 

 - [localize evaluated language abilities](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11199/commits/2af9577ebce3c1dc4ad503cf65d4547b5a19cd5b)
 - [localize evaluated language abilities](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11199/commits/faa578f3225cf4a66acbfc2a619fdc518d3f2717)